### PR TITLE
Update v10.13.0 changelog

### PR DESCRIPTION
**Pull request by ryanml**
### Added
- **[FLASK]** Add Snaps settting to search index ([#14100](https://github.com/MetaMask/metamask-extension/pull/14100))
- Add a new fiat onboarding option via MoonPay ([#13934](https://github.com/MetaMask/metamask-extension/pull/13934))
    - Available for the following networks: Ethereum, BNB Chain, Polygon, Avalanche, Celo
- Add support for a Dark Mode theme ([#14207](https://github.com/MetaMask/metamask-extension/pull/14207))

### Changed
- Improvements for multi-layer fee UX ([#13547](https://github.com/MetaMask/metamask-extension/pull/13547))
    - Fix 'Send max' button when on a multi-layer fee network
    - Show fiat currency estimates alongside ETH estimates on multi-layer fee networks
    - Display L1+L2 gas fees as a combined total on multi-layer fee networks
    - Don't allow users to set gas price on Optimism
- Move Token Detection toggle to Advanced tab. ([#13977](https://github.com/MetaMask/metamask-extension/pull/13977))
- Don’t show ‘What’s new’ pop up to new users ([#13886](https://github.com/MetaMask/metamask-extension/pull/13886))
- Improving settings toggle accessibility by allowing label interaction ([#13876](https://github.com/MetaMask/metamask-extension/pull/13876))
- Updating account and network icons ([#13947](https://github.com/MetaMask/metamask-extension/pull/13947))
- Add 'Enhanced Gas UI' setting to search index ([#14206](https://github.com/MetaMask/metamask-extension/pull/14206))
- Add buy modal link to insufficient currency warning of all networks that have a fiat onramp, and update spacing in the warning's copy ([#14019](https://github.com/MetaMask/metamask-extension/pull/14019))

### Fixed
- Fix issue where editing advanced gas while speeeding up a transaction exits speedup ([#14101](https://github.com/MetaMask/metamask-extension/pull/14101))
- Fix typo in cancel/speed up messaging ([#14067](https://github.com/MetaMask/metamask-extension/pull/14067))
- Fix token icon when going from token detail page to Swaps view ([#14062](https://github.com/MetaMask/metamask-extension/pull/14062))
- Fix issue where the contract address is shown as recipient when calling safe transfer method on erc721 or erc1155 contracts ([#13535](https://github.com/MetaMask/metamask-extension/pull/13535))
- Ensure accounts still line up in dropdown ([#13986](https://github.com/MetaMask/metamask-extension/pull/13986))

### CODE REVIEW

**Review by brad-decker — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [10.13.0]
-### Uncategorized
+### Added
+- **[FLASK]** Add Snaps settting to search index ([#14100](https://github.com/MetaMask/metamask-extension/pull/14100))
+- Add token list aggregattor to token details page ([#14051](https://github.com/MetaMask/metamask-extension/pull/14051))
```
Comment (danjm) [inline CHANGELOG.md:12]: "Add token list aggregattor to token details page" this is still behind feature flag, and so should not be in the changelog yet

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Dark mode: Removing connected svg from account menu in favour of fontawesome ([#14140](https://github.com/MetaMask/metamask-extension/pull/14140))
-- Dark Mode:  Settings -> Networks ([#14134](https://github.com/MetaMask/metamask-extension/pull/14134))
-- Dark Mode:  New Account modal ([#14132](https://github.com/MetaMask/metamask-extension/pull/14132))
-- Dark Mode: Cancel Transaction ([#14129](https://github.com/MetaMask/metamask-extension/pull/14129))
-- Dark Mode: Select Hardware ([#14086](https://github.com/MetaMask/metamask-extension/pull/14086))
-- Bump @metamask/design-tokens from 1.4.2 to 1.4.4 ([#14119](https://github.com/MetaMask/metamask-extension/pull/14119))
-- Sync `master` with `develop` ([#14120](https://github.com/MetaMask/metamask-extension/pull/14120))
-- Add token list to token details page ([#14051](https://github.com/MetaMask/metamask-extension/pull/14051))
-- Dark mode: colors obj UI, GREY, WHITE ([#14118](https://github.com/MetaMask/metamask-extension/pull/14118))
+    - "Send max" doesn't work on Optimism
```
Comment (danjm) [inline CHANGELOG.md:18]: `"Send max" doesn't work on Optimism` -> `Fix 'Send max' button when on the Optimism network`

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+    - Show currency estimate alongside ETH on Optimism
```
Comment (danjm) [inline CHANGELOG.md:19]: `Show currency estimate alongside ETH on Optimism` -> `Show fiat currency estimates alongside ETH estimates on multi-layer fee networks`

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+    - Combine L1+L2 gas when using Optimism
```
Comment (danjm) [inline CHANGELOG.md:20]: `Combine L1+L2 gas when using Optimism` -> `Display L1+L2 gas fees as a combined total on multi-layer fee networks`

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Add labels to provide better accessibility during onboarding v2 ([#13890](https://github.com/MetaMask/metamask-extension/pull/13890))
-- Dark Mode: Fix currently selected network color ([#13918](https://github.com/MetaMask/metamask-extension/pull/13918))
-- Fix yarn.lock ([#13941](https://github.com/MetaMask/metamask-extension/pull/13941))
-- Permissions: convert hook to util ([#13906](https://github.com/MetaMask/metamask-extension/pull/13906))
-- Remove usages of useMetricEvent hook ([#13930](https://github.com/MetaMask/metamask-extension/pull/13930))
-- Show snap version on install ([#13931](https://github.com/MetaMask/metamask-extension/pull/13931))
+
+### Fixed
+- Add 'Enhanced Gas UI' setting to search index ([#14206](https://github.com/MetaMask/metamask-extension/pull/14206))
+- Fixed issue where the "lock screen" shows up briefly between the "create password" screen and the success screen ([#14111](https://github.com/MetaMask/metamask-extension/pull/14111))
```
Comment (danjm) [inline CHANGELOG.md:29]: `Fixed issue where the "lock screen" shows up briefly between the "create password" screen and the success screen` is behind a feature toggle and shouldn't be included

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Add space between sentences in insufficient currency warning ([#14019](https://github.com/MetaMask/metamask-extension/pull/14019))
```
Comment (danjm) [inline CHANGELOG.md:30]: I suggest moving `Add space between sentences in insufficient currency warning` to "Changed" and changing the description to `Add buy modal link to insufficient currency warning of all networks that have a fiat onramp, and update spacing in the warning's copy`

File: CHANGELOG.md [hunk 7]
```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Fix issue where editing advanced gas while speeeding up a transaction exits speedup ([#14101](https://github.com/MetaMask/metamask-extension/pull/14101))
+- Fix typo in cancel/speed up messaging ([#14067](https://github.com/MetaMask/metamask-extension/pull/14067))
+- Fix token icon when going from token detail page to Swaps view ([#14062](https://github.com/MetaMask/metamask-extension/pull/14062))
+- Fix missing insufficient funds error for token sends ([#14026](https://github.com/MetaMask/metamask-extension/pull/14026))
```
Comment (danjm) [inline CHANGELOG.md:34]: `Fix missing insufficient funds error for token sends` was part of a previous release

**Review by danjm — CHANGES_REQUESTED** [review]
Suggested some changes

**Comment by danjm** [comment]
Also, this changelog should include dark mode

**Comment by ryanml** [comment]
Suggested changes have been implemented @danjm

**Review by danjm — APPROVED** [review]

**Pull request merged**